include/crocoddyl/multibody/actuations/floating-base.hpp

```diff
@@ -44,6 +44,14 @@ class ActuationModelFloatingBaseTpl : public ActuationModelAbstractTpl<_Scalar>
     data->tau.tail(nu_) = u;
   };
 
+  virtual void get_actuated(const boost::shared_ptr<Data>& data, Eigen::Ref<VectorXs>& u) {
```

Comment:
This can be obtained with `calc`.
If this is the reason, then we should remove and keep clean the API.

Reply:
get_actuated is the inverse of calc. Look at the implementation before you review.